carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/Screenshot.java

```diff
@@ -59,8 +70,12 @@
 public class Screenshot {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1118)